### PR TITLE
Konstantinos/DPROD-3000/[FE] Revamp Regulatory Information page (ROW)

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

Update Regulatory Page for ROW with Quill Design.

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by dan-deriv — APPROVED** [review]

**Review by hasan-deriv — CHANGES_REQUESTED** [review]
Still old components are here. Need to change those components to quill.

File: src/pages/regulatory/_eu-grid.tsx [hunk 1]
```diff
@@ -1,164 +1,240 @@
...
-                <GridItem name="_t_Italy_t_" url="/regulatory/countries/Italy.pdf" order="13" />
-            </CssGridColumn>
-            <CssGridColumn>
-                <Country>
-                    <Order>14</Order>
-                    <License>
+        <div className={styles.eu_grid}>
+            <CustomLink href="/regulatory/countries/Austria.pdf" className=" inline">
+                <div className={styles.country_container}>
+                    <FlagAustriaIcon className={styles.icon} fill="#000000b8" />
```
Comment (aswathy-deriv) [inline src/pages/regulatory/_eu-grid.tsx:39]: Can you convert this by looping in array as all the components are repeating.We can use an array something like this 
`const countries = [
  { href: "/regulatory/countries/Greece.pdf", icon: <FlagGreeceIcon className={styles.icon} iconSize="md" fill="#000000b8" />, text: "_t_Greece_t_" },
  { href: "/regulatory/countries/Hungary.pdf", icon: <FlagHungaryIcon className={styles.icon} iconSize="md" fill="#000000b8" />, text: "_t_Hungary_t_" },
  { href: "/regulatory/countries/Italy.pdf", icon: <FlagItalyIcon className={styles.icon} iconSize="md" fill="#000000b8" />, text: "_t_Italy_t_" }
];`

File: src/pages/regulatory/index.tsx [hunk 2]
```diff
@@ -1,482 +1,523 @@
...
+                                                    },
+                                                    title: 'Key information documents',
+                                                },
+                                                {
+                                                    className: `${styles.accordion_border}`,
+                                                    content: function noRefCheck() {
+                                                        return (
+                                                            <div className="flex lg:flex-row items-start justify-start flex-col ">
+                                                                <CustomLink
+                                                                    href="/regulatory/RTS28-2022.pdf"
```
Comment (aswathy-deriv) [inline src/pages/regulatory/index.tsx:120]: Similarly here by looping via array if the components are repeating

**Review by aswathy-deriv — DISMISSED** [review]

File: src/pages/regulatory/eu_grid.module.scss
Comment (hasan-deriv) [inline src/pages/regulatory/eu_grid.module.scss:1]: Can you try only responsive css to move in scss module and rest of them in tailwind?

**Comment by konstantinosG-deriv** [comment]
Opening new PR with resolved conflicts

**Pull request closed without merging**